tracer/test/snapshots/Security.AspNetCore2.__test=blocking_enableSecurity=True_expectedStatusCode=403_url=_.verified.txt

```diff
@@ -27,7 +27,7 @@
       runtime-id: Guid_1,
       span.kind: server,
       _dd.appsec.event_rules.version: 1.3.0,
-      _dd.appsec.json: {"triggers":[{"rule":{"id":"ublock","name":"Hello","tags":{"category":"attack_attempt","type":"security_scanner"}},"rule_matches":[{"operator":"match_regex","operator_value":"Hello\\/v","parameters":[{"address":"server.request.headers.no_cookies","highlight":["Hello/V"],"key_path":["user-agent",0],"value":"Mistake Not... Hello/V"}]}]}]},
+      _dd.appsec.json: {"triggers":[{"rule":{"id":"ublock","name":"Hello","tags":{"category":"attack_attempt","type":"security_scanner"}},"rule_matches":[{"operator":"match_regex","operator_value":"Hello\\/v","parameters":[{"address":"server.request.headers.no_cookies","highlight":["Hello/V"],"key_path":["user-agent","0"],"value":"Mistake Not... Hello/V"}]}]}]},
```

Comment:
Just checking, is the change in `user-agent` from a number to a string correct?

Reply:
yes indeed, it's been something that changed with waf 1.5 and that has been changed accordingly in the system tests